pkg/adminconfig/rego_file_evaluator.go

```diff
@@ -69,9 +71,20 @@ func (r *RegoPolicyEvaluator) prepareQuery() (rego.PreparedEvalQuery, error) {
 	if err != nil {
 		return rego.PreparedEvalQuery{}, errors.Wrap(err, "couldn't compile modules")
 	}
+	content, err := ioutil.ReadFile("/tmp/adminconfig/infrastructure.json")
```

Comment:
There is a constant for "/tmp/adminconfig". Please use it.

Reply:
Changed